oclint-driver/include/oclint/ConfigFile.h

```diff
@@ -40,8 +40,12 @@ class ConfigFile
     int _maxP1;
     int _maxP2;
     int _maxP3;
+    TriState _listEnabledRules = UNDEFINED;
     TriState _clangChecker = UNDEFINED;
     TriState _allowDuplicatedViolations = UNDEFINED;
+    TriState _enableGlobalAnalysis = UNDEFINED;
+    TriState _noAnalytics = UNDEFINED;
+    TriState _enableVerbose = UNDEFINED;
```

Comment:
I can see it makes sense to add `_enableGlobalAnalysis` as I can see people want to save that setting. And let's remove the others.